packages/playground/blueprints/src/lib/steps/common.ts

```diff
@@ -52,6 +52,23 @@ export async function fileToUint8Array(file: File) {
  * Import the polyfilled File class below to ensure its buffer is available to
  * functions like writeFile (./client-methods.ts) and fileToUint8Array (above).
  */
+
+globalThis.File =
+	globalThis.File ||
+	class extends Blob {
+		lastModified = 0;
+		constructor(
+			bits: any,
+			name: string,
+			options: { type?: string; lastModified?: number }
+		) {
+			super(bits);
+			if (options.lastModified) {
+				this.lastModified = options.lastModified;
+			}
+		}
+	};
+
```

Comment:
Is this change required by the Blueprints builder? Polyfilling the File class seems like a distinct issue.

Reply:
@adamziel I originally figured we could roll that update into this one, but you're right, we need to keep it separate. I've got another branch for that now.